examples/ecs-firelens/fluent-bit.conf

```diff
@@ -0,0 +1,41 @@
+[SERVICE]
+    Log_Level debug
```

Comment:
nit: I'd set log level to info in an example

Reply:
removed service section